mods/default/locale/default.uk.tr

```diff
@@ -0,0 +1,211 @@
+# textdomain: default
+Locked Chest =Замкнена скриня
+Locked Chest (owned by @1) = Замкнена скриня (належить @1)
+You do not own this chest. = Ви не володієте цією скринею.
+a locked chest = замкнена скриня
+Chest = Скриня
+Stick = Палиця
+Paper = Папір
+"@1" by @2 = "@1" від @2
+Book = Книга
+Book with Text = Книга з текстом 
+Skeleton Key = Скелетний ключ
+Key to @1's @2 = Ключ до @2 від @1
+Coal Lump = Вугільний грудочок
+Iron Lump = Залізний грудочок
+Copper Lump = Мідний грудочок
+Tin Lump = Олов'яний грудочок
+Mese Crystal = Кристал Мезе
+Gold Lump = Золотий грудочок
+Diamond = Алмаз
+Clay Lump = Глиняний грудочок
+Steel Ingot = Сталевий злиток
+Copper Ingot = Мідний злиток
+Tin Ingot = Олов'яний злиток
+Bronze Ingot = Бронзовий злиток
+Gold Ingot = Золотий злиток
```

Comment:
same here and in other files

Reply:
After the translation is complete, we will have to walk again